### PR TITLE
sample integration with sentry (#703)

### PR DESCRIPTION
**Pull request by speedcm**
Setup sample integration with Sentry.

### CODE REVIEW

**Comment by speedcm** [comment]
This DevOps effort is tracked via the GitHub issue https://github.com/skylight-hq/ctoec-devops/issues/14.

To setup Sentry in the application, the following source files were modified (identified because there were numerous other dummy commits to test Sentry integration):
 
* Hedwig.csproj
  * PackageReference Include="Sentry.AspNetCore" Version="2.0.1" 
  * PackageReference Include="Sentry.Extensions.Logging" Version="2.0.1"
* Program.cs
  * logging.Services.Configure<Sentry.Extensions.Logging.SentryLoggingOptions>(context.Configuration.GetSection("Sentry"));
  * logging.AddSentry();
  * .UseSentry()
* appsettings.json file
   * See the Sentry stanza

The azure-pipelines.yml file was only modified to allow for artifact packaging for a PR build.  The changes made were re-set back to original.

**Comment by speedcm** [comment]
* Developer Notes
  * This setup supports the backend
     * Logging errors and above to report to Sentry
     * Application Exceptions to report to Sentry
  * This setup does not support the client side
     * Sentry information needs to be injected into a property file

**Comment by speedcm** [comment]
Prior to merging this PR, the Azure Pipelines for Staging and Production need to be updated to include Sentry integration.

**Comment by speedcm** [comment]
For more information on Sentry, see the DevOps documented procedures: https://github.com/skylight-hq/ctoec-devops/blob/master/docs/TOOLS-SENTRY.md

File: src/Hedwig/Program.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public static void Main(string[] args)
 			var host = CreateHostBuilder(args).Build();
 			var environment = GetEnvironmentNameFromAppSettings();
 
-			if(environment != Environments.Production) {
+				if (environment != Environments.Production) {
```
Comment (jlhogan) [inline src/Hedwig/Program.cs]: nit but this does not seem like correct indentation :)

**Comment by speedcm** [comment]
The indentation is now fixed. Ty!

**Pull request merged**